src/doc/selected_layers.cpp

```diff
@@ -99,6 +99,24 @@ LayerList SelectedLayers::toBrowsableLayerList() const
   return output;
 }
 
+LayerList SelectedLayers::toAllTilemaps() const
```

Comment:
Probably I'd change the name to
```suggestion
LayerList SelectedLayers::onlyTilemaps() const
```

Reply:
I used this name because there are toBrowsableLayerList and toAllLayersList methods already, so I tried to keep it consistent...let me know if I should rename it to onlyTilemaps anyway

Reply:
Seeing that it returns a `LayerList` (and not a filtered/new `SelectedLayers`) it can be called `toAllTilemaps()` (I think all 3 names are confusing anyway, probably something to rename in the future).